### PR TITLE
adding ribbons to bid tracker

### PR DESCRIPTION
**Pull request by SidelineCory24**
This is only a styling PR. Still need to verify what needs to be hooked into on the BE for all of the ribbons.

### CODE REVIEW

**Comment by SidelineCory24** [comment]
Need to hookup ribbons

**Comment by SidelineCory24** [comment]
This is only a styling update. Still need to verify what you should be hooking into on the BE.

File: src/Components/ProfileDashboard/BidList/BidList.jsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ const BidList = ({ bids, submitBidPosition, deleteBid, registerHandshake, isLoad
       readOnly={isPublic}
       userId={userId}
       useCDOView={useCDOView}
+      showRibbons={false}
```
Comment (elizabeth-jimenez) [inline src/Components/ProfileDashboard/BidList/BidList.jsx:38]: Nice!

**Comment by elizabeth-jimenez** [comment]
image
1280*800 :(

**Comment by SidelineCory24** [comment]
Issue occurs if the `Submitted by CDO` text is gone but not if it's there for 1280x800. Will look into

**Comment by elizabeth-jimenez** [comment]
> Issue occurs if the `Submitted by CDO` text is gone but not if it's there for 1280x800. Will look into

I think collapsing the text to HTF, CN, SND at the smaller screen sizes could work. Like how we collapse it to just an icon on the Compare page.

File: src/Components/MediaQuery/MediaQuery.jsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const MediaQueryWrapper = ({ breakpoint, widthType, children }) => {
 };
 
 MediaQueryWrapper.propTypes = {
-  breakpoint: PropTypes.oneOf(['screenSmMin', 'screenXsMax', 'screenMdMin', 'screenSmMax', 'screenLgMin', 'screenMdMax']).isRequired,
+  breakpoint: PropTypes.oneOf(['screenSmMin', 'screenXsMax', 'screenMdMin', 'screenSmMax', 'screenLgMin', 'screenMdMax', 'screenXlgMin', 'screenXlgMax']).isRequired,
```
Comment (elizabeth-jimenez) [inline src/Components/MediaQuery/MediaQuery.jsx:19]: i think the last one is supposed to be `screenLgMax`?

File: src/sass/sass-vars/variables.js [hunk 3]
```diff
@@ -18,6 +18,12 @@ module.exports = {
   get screenMdMaxNum() { return this.screenLgMinNum - 1; },
   get 'screen-md-max'() { return `${this.screenMdMaxNum}px`; },
 
+  screenXlgMinNum: 1400,
+  get 'screen-xlg-min'() { return `${this.screenXlgMinNum}px`; },
+
+  get screenXLgMaxNum() { return this.screenXlgMinNum - 1; },
```
Comment (elizabeth-jimenez) [inline src/sass/sass-vars/variables.js:24]: `screenLgMaxNum`?

Comment (elizabeth-jimenez) [reply src/sass/sass-vars/variables.js:24]: we did set it up properly earlier, but we didn't name it correctly 😅 
Line 24 should be: `get screenLgMaxNum() { return this.screenXlgMinNum - 1; },`

**Review by elizabeth-jimenez — APPROVED** [review]
Just the variable name, but besides that: 🍰 🍬 🍭 👍

File: src/sass/sass-vars/variables.js (hunk 3 above)
Comment (SidelineCory24) [reply src/sass/sass-vars/variables.js:24]: done

File: src/Components/MediaQuery/MediaQuery.jsx (hunk 2 above)
Comment (SidelineCory24) [reply src/Components/MediaQuery/MediaQuery.jsx:19]: done

**Pull request merged**